bitnami/redis/templates/scripts-configmap.yaml

```
@@ -109,7 +109,7 @@ data:
         fi
 
         info "about to run the command: $sentinel_info_command"
-        retry_while "eval $sentinel_info_command" 2 1
+        retry_while "eval $sentinel_info_command" 2 5
```

Would setting it to retry 5 times, 1 second each, avoid the drawback? 
```suggestion
        retry_while "eval $sentinel_info_command" 5 1
```

---

@fevisera I'm wary of increasing the number of retries - it may trigger [this issue](https://github.com/bitnami/charts/issues/17866) again.  IMO it's safer to only retry once, but sleep 5s between retries and avoid a dead loop.  Note that I've also reduced [sentinel.getMasterTimeout](https://github.com/bitnami/charts/pull/20612/files#diff-124e7200c9157afc7c5dcfbf90f04932771624080855e564889d2b21ef64ee3fR366) to take the increased sleep time into account (though perhaps reducing by 9s was too aggressive - what do you think?)

---

Thank you for your detailed explanation and for striving to strike a balance between security and efficiency in the configuration. LGTM!